bokehjs/src/lib/models/transforms/jitter.ts

```diff
@@ -35,10 +36,28 @@ export class Jitter extends RangeTransform {
   }
 
   override v_compute(xs0: Arrayable<number | Factor>): Arrayable<number> {
-    if (this.previous_values != null && this.previous_values.length == xs0.length)
-      return this.previous_values
-    this.previous_values = super.v_compute(xs0)
-    return this.previous_values
+    let xs: Arrayable<number>
+    if (super.range instanceof FactorRange)
+      xs = super.range.v_synthetic(xs0)
```

Comment:
`range` attribute is inherited, so there's no reason to use `super` here. Just use `this`.

Reply:
Changed to `this`